packages/core/src/dev/activation.ts

```diff
@@ -160,6 +160,11 @@ export async function activate(ctx: vscode.ExtensionContext): Promise<void> {
             webviewOptions: {
                 retainContextWhenHidden: true,
             },
+        }),
+        vscode.commands.registerCommand('aws.explorer.setLoginService', (serviceToShow?: string) => {
+            if (toolkitAuthProvider.webView && 'setLoginService' in toolkitAuthProvider.webView.server) {
+                toolkitAuthProvider.webView.server.setLoginService(serviceToShow)
```

Comment:
how will this be reset from code catalyst, if the user starts using Q IdC for example

Reply:
This is reset everytime command `aws.${prefix}.auth.manageConnections` is triggered. This command runs whenever use clicks Sign-in button in toolkit or Add New Connection in Toolkit, which renders the LoginView.

Reply:
Ideally I should pass the code catalyst as a param into the webview but I did not find out an easy way.

Reply:
In the current UX, I have yet found out a way to do below flow

>the user starts using Q IdC 
> the user re-use this connection for CodeCatalyst

This flow currently does not exist. We can revisit this once we got more feedback.